storage/posix/files.go

```diff
@@ -591,7 +591,6 @@ func (a *appender) publishCheckpoint(minStaleness time.Duration) error {
 	if err := a.s.overwrite(layout.CheckpointPath, cpRaw); err != nil {
 		return fmt.Errorf("overwrite(%s): %v", layout.CheckpointPath, err)
 	}
-	klog.Infof("Published latest checkpoint")
```

Comment:
Rather than remove it entirely, how about making it a `klog.V(1).Infof` (or even `V(2)`) and include the tree size in the logged line?

Reply:
AWS: https://github.com/transparency-dev/trillian-tessera/blob/a60c8d23e801e11eb7511a623d144d6a7da3d0d4/storage/aws/aws.go#L760

GCP: https://github.com/transparency-dev/trillian-tessera/blob/a60c8d23e801e11eb7511a623d144d6a7da3d0d4/storage/gcp/gcp.go#L629

Reply:
The same info logging cannot be found in AWS and GCP storage implementation of `publishCheckpoint`.

Reply:
^ was the reason I chose to delete it, and there's already logging for when a new tree state is published with the new tree size - https://github.com/transparency-dev/trillian-tessera/blob/a60c8d23e801e11eb7511a623d144d6a7da3d0d4/storage/posix/files.go#L346

Reply:
Integration != Publishing in Tessera, and in fact these are asynchronous processes.
 
Tessera is not yet at a point in its maturity cycle where I'm happy for debug logging to be entirely removed, if there's a disparity between events logged in different storage backends I'd argue for _adding_ similar logging in the other storage implementations as opposed to removing, as @roger2hk's comment above also suggests.

I _am_ happy for debug logging to have its level moved up so that it can be enabled/disabled, which was the suggestion.

Reply:
@AlCutter Added logging for checkpoint publishing for all backends.

Reply:
Ace, thanks!